comictaggerlib/cli.py

```diff
@@ -372,14 +378,19 @@ def process_file_cli(
 
         # now, search online
         if opts.online:
+            sources_api = SourcesAPI()
+            if sources_api.sources[0]["talker"].source_details["static_options"]["has_issues"]:
+                match_results.has_issues = True
             if opts.issue_id is not None:
-                # we were given the actual ID to search with
+                # we were given the actual ID to search with. This can now be a series id too!
```

Comment:
Not going to happen. Options need to be consistent issue_id will need to stay only the issue. We can add a volume_id option later.

Reply:
Shall I revert the whole thing or leave in the source checks to see if it supports issues? If not error out?

Looking ahead. The CLI is "--id" so should that be changed to "issue-id"? Leave until the setting change comes in?

Reply:
Eventually I will add an alias so as not to break existing use-cases, so it will have both options. For now let's remove the source checks.